app/controllers/follows_controller.rb

```diff
@@ -0,0 +1,24 @@
+class FollowsController < ApplicationController
+  before_action :authenticate_user!
+  load_and_authorize_resource
+
+  def create
+    @followable = find_followable
+    @follow = Follow.create(user: current_user, followable: @followable)
+    render :refresh_follow_button
+  end
+
+  def destroy
+    @follow = Follow.find(params[:id])
+    @followable = @follow.followable
```

Comment:
I think I'm missing something, and it may be my ignorance about destroy usage. Where would be used the `@followable` variable?